### PR TITLE
add global price to the charts

### PR DESCRIPTION
**Pull request by p6te**
Add global price to:
- position details
- adding new pool
- adding new position

#### Global price should be included in handling the refresh button after this merge. 

image

image

image

changed styles of shadows for current and global price:

image

### CODE REVIEW

File: src/components/Inputs/SimpleInput/SimpleInput.tsx [hunk 1]
```diff
@@ -75,6 +77,24 @@ export const AmountInput: React.FC<IProps> = ({
...
+              <Typography className={classes.textGlobalPrice} variant='caption'>
+                Global price
+              </Typography>
+            }
+            placement='right-start'
+            classes={{
+              tooltip: classes.globalPriceTooltip
+            }}>
+            <Typography className={classes.globalPrice} variant='h4'>
+              {globalPrice > 1 ? globalPrice.toFixed(2) : globalPrice}
```
Comment (wojciech-cichocki) [inline src/components/Inputs/SimpleInput/SimpleInput.tsx:93]: Change precision to `0.0001. This accuracy is not only intended for prices below 1. Consider, for example, the case of stablecoins: The ratio is displayed with high precision only in one direction, upon inrversal, the ratio is already rounded to `10^(-2)`

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Change precision of `0.0001`

File: src/components/Inputs/SimpleInput/SimpleInput.tsx (hunk 1 above)
Comment (p6te) [reply src/components/Inputs/SimpleInput/SimpleInput.tsx]: fix applied

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Is it possible to play with the css styles so that both prices are visible when they are close? Currently, only the current price is visible even with zoom-in

**Review by wojciech-cichocki — COMMENTED** [review]
It's worth mentioning that the global price should be updated after clicking the refresh button. So depending on the merging order in one of those tasks, this needs to be taken into account

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
The current issue here is the shadow on the "current price". Is there possibility to make the shadow smoother at the top? Currently, it looks quite pixelated because of this:
image

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**